services/auth/src/azure.js

```diff
@@ -18,7 +19,7 @@ builder.declare({
   name: 'azureAccounts',
   input: undefined,
   output: 'azure-account-list-response.yml',
-  stability: 'stable',
+  stability: APIBuilder.stability.stable,
```

Comment:
Should this be deprecated?

Reply:
discussed offline, yes, switching to deprecated. This endpoint just returned existing configurations, which could be done with secrets service